services/storage/src/simcore_service_storage/modules/celery/worker.py

```diff
@@ -1,28 +1,34 @@
 import logging
 
-from celery import Celery  # type: ignore[import-untyped]
+from celery import Celery, Task  # type: ignore[import-untyped]
+from common_library.async_tools import make_async
 from models_library.progress_bar import ProgressReport
 from servicelib.logging_utils import log_context
 
-from .models import TaskID
-
 _logger = logging.getLogger(__name__)
 
 
-class CeleryTaskQueueWorker:
+class CeleryWorkerClient:
```

Comment:
why client? this is confusing. in celery you have a concept of a Celery Application (or client) and then of Celery Workers. so please let's keep it at that, let's not have clients everywhere.